javascriptv3/example_code/cross-services/lambda-scheduled-events/src/helper-functions/lambda-function-setup.ts

```diff
@@ -0,0 +1,61 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+
+ABOUT THIS NODE.JS EXAMPLE: This example works with AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3. This example is in the 'AWS SDK for JavaScript v3 Developer Guide' at
+https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/invoke-lambda-functions-with-scheduled-events.html.
+
+Purpose:
+lambda-function-setup.ts demonstrates how to create an AWS Lambda function.
+It is part of a tutorial demonstrates execute Lambda functions using Amazon CloudWatch scheduled events. To see the full tutorial, see
```

Comment:
tutorial **that** demonstrates **how to run** Lambda
(fix typos, plus use 'run' instead of 'execute')